runtime/lua/vim/lsp/rpc.lua

```diff
@@ -1,3 +1,4 @@
+local vim = vim
```

Comment:
Why did you need to add this line?

Reply:
It brings vim into the local scope instead of the global scope which is more efficient. Looking up a global value means hitting the global table, whereas a local value is an upvalue captured by the functions. It also hides a lot of the errors I get from my Lua LSP.

Reply:
It's a micro-optimization.